l1-contracts/test/portals/UniswapPortal.t.sol

```diff
@@ -66,22 +66,39 @@ contract UniswapPortalTest is Test {
     deal(address(DAI), address(daiTokenPortal), amount);
   }
 
-  // L2 to L1 message to be added to the outbox
-  function _createDaiWithdrawMessage(address _recipient) internal view returns (bytes32 entryKey) {
+  // L2 to L1 message to be added to the outbox,
+  // by default recipient is the uniswap portal and we set a designated caller
+  function _createDefaultDaiWithdrawMessage() internal view returns (bytes32 entryKey) {
+    return _createDaiWithdrawMessage(address(uniswapPortal), true);
+  }
+
+  function _createDaiWithdrawMessage(address _recipient, bool _withCaller)
```

Comment:
When you use other "wrappers" for this function, seem like this could be remade to be a function that has args specifying everything instead of using some constants. Not vital, but makes it easier to figure out what is going on if you don't have to step through multiple functions to find the one doing things 👀